Midtrans/Config.php

```diff
@@ -66,8 +66,10 @@ class Config
      */
     public static $curlOptions = array();
 
-    const SANDBOX_BASE_URL = 'https://api.sandbox.midtrans.com';
-    const PRODUCTION_BASE_URL = 'https://api.midtrans.com';
+    const SANDBOX_BASE_URL_V1 = 'https://api.sandbox.midtrans.com/v1';
+    const PRODUCTION_BASE_URL_V1 = 'https://api.midtrans.com/v1';
+    const SANDBOX_BASE_URL = 'https://api.sandbox.midtrans.com/v2';
+    const PRODUCTION_BASE_URL = 'https://api.midtrans.com/v2';
     const SNAP_SANDBOX_BASE_URL = 'https://app.sandbox.midtrans.com/snap/v1';
     const SNAP_PRODUCTION_BASE_URL = 'https://app.midtrans.com/snap/v1';
```

Comment:
After releasing hotfix, please improve the implementation:

According to docs https://api-docs.midtrans.com/#https-request base URL does not include /v1 or /v2. So let's follow docs. Base url is without `/v1`.

The additional `/v1, /v2` on url path should be added on each Function that call the HTTP client, like here:
https://github.com/Midtrans/midtrans-go/blob/7889ad6bdcf9e7fd1f5565fd74f2e81e86f53099/coreapi/clientsubscription.go#L115-L121